src/owasp-dependency-check-suppressions.xml

```diff
@@ -37,6 +37,23 @@
         <vulnerabilityName regex="true">.*</vulnerabilityName>
     </suppress>
 
+    <suppress>
+        <notes><![CDATA[
+       file name: snakeyaml-1.32.jar
```

Comment:
@lhotari IIRC you have some previous work on this dependency. Could you refer to that PR here?